src/oauth-tooling.ts

```diff
@@ -246,6 +245,28 @@ function getAccessToken(options: OAuthConfig, logger?: Logger): Promise<Token> {
   });
 }
 
+function getCredentials(options: OAuthConfig) {
```

Comment:
Should we switch to arrow functions, that was also part of refactoring in other parts.